### PR TITLE
Section: hp-Interface

### PR DESCRIPTION
**Pull request by marcfehling**

### CODE REVIEW

File: 9.2/paper.tex [hunk 1]
```diff
@@ -446,7 +446,61 @@ \subsection{Improved large-scale performance}
...
-\todo[inline]{Marc: Your section}
+With the previous release, \dealii supports $hp$-adaptive finite element methods
+on distributed memory systems \cite{dealII91}. We implemented the bare functionality
+for $hp$-adaptive methods with the objective to offer the greatest flexibility in
+their application. Here, reference finite elements still had to be assigned manually
+to each cell, which may not lead to an optimal mesh and is tedious.
+
+With the current release, $hp$-adaptive finite element methods have been further
+expanded in both the range of features in terms of decision strategies and their 
+user interface, effectively making the more attractive to use. We introduced many
```
Comment (kronbichler) [inline 9.2/paper.tex:457]:
```suggestion
user interface, effectively making them more attractive to use. We introduced many
```

File: 9.2/paper.tex [hunk 2]
```diff
@@ -446,7 +446,61 @@ \subsection{Improved large-scale performance}
...
 
-\todo[inline]{Marc: Your section}
+With the previous release, \dealii supports $hp$-adaptive finite element methods
+on distributed memory systems \cite{dealII91}. We implemented the bare functionality
+for $hp$-adaptive methods with the objective to offer the greatest flexibility in
+their application. Here, reference finite elements still had to be assigned manually
+to each cell, which may not lead to an optimal mesh and is tedious.
+
+With the current release, $hp$-adaptive finite element methods have been further
+expanded in both the range of features in terms of decision strategies and their
```
Comment (kronbichler) [inline 9.2/paper.tex:456]:
```suggestion
expanded in both the range of features like decision strategies and their 
```
? Here I'm not sure what you want to say.

File: 9.2/paper.tex [hunk 3]
```diff
@@ -446,7 +446,61 @@ \subsection{Improved large-scale performance}
...
+current error estimates and adaptation flags. In the next refinement cycle, these
+predicted error estimates allow us to decide whether the choice of adaptation in
+the previous cycle was justified, and provide the foundation for it in the next
+one \cite{MelenkWohlmuth2001}.
+
+In general, $p$-refinement is favorable over $h$-refinement in smooth regions of
+the finite element approximation \cite[Thm.~3.4]{BabuskaSuri1990}. Thus, estimating
+its smoothness provides a suitable decision indicator for $hp$-adaptation. For this
+purpose, we expand the finite element approximation into a series of orthogonal
+basis functions of increasing complexity, and consider the decay of their expansion
```
Comment (kronbichler) [inline 9.2/paper.tex:500]: Would you be fine with changing this into
```suggestion
purpose, we express the finite element approximation in an orthogonal
basis of increasing frequency, and consider the decay of their expansion
```
I do not really understand the "series" here (from a mathematical point of view), and I do not think that complexity is the right word here. You either use Legendre or Fourier (or is there other choices?), both of which you could attribute as being decomposed by some sort of "frequency".

Comment (kronbichler) [reply 9.2/paper.tex:500]: Another term used in the literature of high-order methods is "oscillatory" for the higher-order corrections, which fits both Legendre and Fourier.

File: 9.2/paper.tex (hunk 2 above)
Comment (marcfehling) [reply 9.2/paper.tex]: I made two sentences out of this one. Hope that helps :)

File: 9.2/paper.tex (hunk 3 above)
Comment (marcfehling) [reply 9.2/paper.tex]: 'series' as meant in 'series expansion'. I like your suggestions with the term 'frequency'. This brings it to the point.

**Pull request merged**